contracts/art_nft_psp34/lib.rs

```diff
@@ -48,19 +41,35 @@ pub mod art_nft_psp34 {
                 _ => panic!("Failed to set default base_uri")
             }
         }
+
+        /// Mint one NFT for each given account, using TokenID U64(0..n)
```

Comment:
The comment is a bit misleading because one could understand that each call would restart token ID from 0 while it actually continues starting from `total_supply` (which is 0 initially). Furthermore, it seems that some implementors use ID 0 as the collection ID so maybe we should start token IDs from 1 and keep 0 unused to prevent any misinterpretation of token 0's metadata.

Reply:
Regarding the comment, it's indeed misleading. I will change.

Reply:
Regarding ID's used to store metadata, when following these convention, there is no confusion:
`U64(n)` is used for token id's, while `U8(0)` is indeed sometimes used for collection metadata.